frontend/cypress/e2e/05-change-password/spec.cy.js

```diff
@@ -1,5 +1,126 @@
+  });
+
+  it('Verify changing password with password reuse', () => {
+    cy.login(email, 'New12!')
+    cy.confirmPassword(validPassword, validPassword)
+    cy.contains('div', 'Password has been used').should('be.visible');
+  });
+
+  it('Verify changing password with password missing uppercase letter', () => {
+    cy.login(email, validPassword)
+    cy.confirmPassword('mynewpw846!', 'mynewpw846!')
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password missing numbers', () => {
+    cy.login(email, 'mynewpw846!')
+    cy.confirmPassword('DoeJane!', 'DoeJane!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password missing lowercase letter', () => {
+    cy.login(email, 'DoeJane!')
+    cy.confirmPassword('DOEJ284!', 'DOEJ284!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password missing special character', () => {
+    cy.login(email, 'DOEJ284!')
+    cy.confirmPassword('DoeJ837', 'DoeJ837');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password has sequential characters', () => {
+    cy.login(email, 'DoeJ837')
+    cy.confirmPassword('Abcd1234!', 'Abcd1234!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password has repeated characters', () => {
+    cy.login(email, 'Abcd1234!')
+    cy.confirmPassword('Pass1111!', 'Pass1111!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password contains user information (user name)', () => {
+    cy.login(email, 'Pass1111!')
+    cy.confirmPassword('JaneDoe1936!', 'JaneDoe1936!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify changing password with password contains user email', () => {
+    cy.login(email, 'Pass1111!')
+    cy.confirmPassword('JaneDoe1936!', 'JaneDoe1936!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+  
+  it('Verify changing password with common password', () => {
+    cy.login(email, 'JaneDoe1936!')
+    cy.confirmPassword('Password1!', 'Password1!');
+    cy.contains('div', 'Invalid password').should('be.visible');
+  });
+
+  it('Verify canceling password change process', () => {
+    cy.login(email, 'Password1!')
+    cy.confirmPassword(validPassword, validPassword);
+    cy.get('.navbar-brand').click()
+    cy.contains('div', 'Profile updated successfully').should('not.be.visible')
+  });
+
+  
+  it('Verify blank new password field', () => {
+    cy.login(email, validPassword)
+    cy.confirmPassword(validPassword);
+    cy.get('.navbar-brand').click()
+    cy.contains('div', 'Confirm password can not be blank').should('be.visible')
+  });
```

Comment:
này theo trên web là nó check Password do not match á chắc chị sửa lại nhe, vẫn có thể cho TC này pass đc á tại validate confirm password thì nó chỉ cần check giống khác new password thoi chứ có required hay hem thì này tuỳ business rule é

Reply:
okie nha